starter/package-lock.json

```diff
@@ -1456,9 +1725,8 @@
       }
     },
     "@nationalbankbelgium/stark-build": {
-      "version": "10.0.0-beta.3",
-      "resolved": "https://registry.npmjs.org/@nationalbankbelgium/stark-build/-/stark-build-10.0.0-beta.3.tgz",
-      "integrity": "sha512-zMm5dmH73SvqFN0eFQGFLEZzxZq8tR4Q0UZp+y0+ky+GcC7OjjNf8aBikxvrcnuvMeynFqZJJsCB9VW570rVmQ==",
+      "version": "file:../dist/packages-dist/stark-build/nationalbankbelgium-stark-build-10.0.0-beta.4-f331991e.tgz",
```

Comment:
Can you revert these 2 lines? They should point to the latest version of Stark on NPM ;)